modules/ROOT/pages/con_tasks-syntax.adoc

```diff
@@ -0,0 +1,74 @@
+[]
+= Tasks Syntax
+
+To define a configuration file for a `Task` resource, specify the
+following fields:
+
+* Required:
+  * `apiVersion` - Specifies the API version.
+  * `kind` - Specify the `Task` resource object
+  * `metadata` - Specifies data to uniquely identify the
+    `Task` resource object, for example a `name`.
+  * `spec` - Specifies the configuration information for
+    your `Task` resource object. `Task` steps must be defined through either of
+    the following fields:
+  ** link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#steps[`steps`] - Specifies one or more container images that you want
+      to run in your `Task`.
+
+* Optional:
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#inputs[`inputs`] - Specifies parameters and `PipelineResources` needed by your `Task`. A `Task` can declare the inputs it needs, which can be either or both of:
+
+** link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#parameters[`parameters`]
+** link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#input-resources[input resources] 
+ 
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#steps[`outputs`] - Specifies `PipelineResources`
+    created by your `Task`
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#volumes[`volumes`] - Specifies one or more volumes that you want to make
+    available to your `Task`'s steps.
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#step-template[`stepTemplate`] - Specifies a `Container` step
+    definition to use as the basis for all steps within your `Task`.
+
+
+The following example shows a YAML file where the configuration fields are completed:
+
+[source,yaml]
+----
```

Comment:
Here you have specified the source. The same way specify the source in other code snippets in this PR.